### PR TITLE
Add initial file structure

### PR DESCRIPTION
**Pull request by joeylouise**
This PR sets up the initial file structure, including following folders and files:

- public
- src
- database_build
- tests
- eslint
- package.json

Relates #1

### CODE REVIEW

File: public/scripts/d3/margins.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/* eslint-disable */
+
+var width = 1000;
```
Comment (des-des) [inline public/scripts/d3/margins.js]: Width here has global scope

You need to decide how you are going to modularize your code.

Using an iife is the easiest option, happy to go into more detail if you like

File: public/main.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  align-items: center;
+  background: lightgrey;
+}
+
+#d3-canvas {
+  width: 360px;
+  height: 640px;
+}
+
+svg {
```
Comment (des-des) [inline public/main.css]: You want to apply this to all svgs?

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="main.css">
+    <title>Echo</title>
+  </head>
+  <body>
+    <div id="desktop-background" >
+      <div id="d3-canvas">
+
```
Comment (des-des) [inline public/index.html]: Hmm, are you planning on using a canvas element?

A canvas is a html5 thing that I expect you will not use. The name is misleading

File: public/scripts/d3/margins.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+var height = 1000;
+
+var svg = d3.select("#d3-canvas")
+  .append("svg")
+    // .attr("background-color", "pink")
```
Comment (des-des) [inline public/scripts/d3/margins.js]: No comments please

Comment (des-des) [reply public/scripts/d3/margins.js]: I mean commented out code. If you want to merge some commented code into master, then leave a comment to explain why it needs to stay

**Review by des-des — CHANGES_REQUESTED** [review]

**Comment by des-des** [comment]
Final thing, try to add issue numbers to *all* commits, not sure I might have forgot to mention that

File: public/scripts/d3/margins.js (hunk 1 above)
Comment (ghost) [reply public/scripts/d3/margins.js]: Hi, we wanted to give width/heigh etc global scope as it's likely we'll need to refer to it later on in our d3 code – if we use an iife I don't think we'll have access to the variable, right?

**Review by des-des — APPROVED** [review]

**Pull request merged**